Update versioning-sdk-msbuild-vs.md (#27850)
dotnet/docs: docs/core/porting/versioning-sdk-msbuild-vs.md

| 2.1.8xx          | 16.2 (No VS)       | July '19     | Aug '21   |
| 3.1.1xx          | 16.4               | Dec '19      | Oct '21   |
| 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
| 5.0.2xx          | 16.9               | March '21    | Aug '22   |
| 5.0.3xx          | 16.10              | May '21      | Aug '21   |
| 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |

> | 6.0.100          | 17.0**             | Nov '21      | tbd       |

Reviewer: ```suggestion
| 6.0.100          | 17.0**             | Nov '21      | TBD       |
```

> | 6.0.200          | 17.1               | Feb '22      | tbd       |

Reviewer: ```suggestion
| 6.0.200          | 17.1               | Feb '22      | TBD       |
```

> | 6.0.300          | 17.2^              | tbd          | tbd       |

Reviewer: ```suggestion
| 6.0.300          | 17.2^              | TBD          | TBD       |
```

| 2.1.8xx          | 16.2 (No VS)       | July '19     | Aug '21   |
| 3.1.1xx          | 16.4               | Dec '19      | Oct '21   |
| 3.1.4xx          | 16.7               | Aug '20      | Dec '22   |
| 5.0.1xx          | 16.8               | Nov '20      | Mar '21   |
| 5.0.2xx          | 16.9               | March '21    | Aug '22   |
| 5.0.3xx          | 16.10              | May '21      | Aug '21   |
| 5.0.4xx          | 16.11              | Aug '21      | Feb '22*  |

> | 6.0.100-rc1      | 17.0-preview 4     | Sep '21      | N/A       |

Reviewer 2: please remove 6.0.100-rc1

Author: done

> | 6.0.100          | 17.0**             | Nov '21      | May '23   |

Reviewer 2: For these three, please remove the May dates. Those are all dependent on when 17.2 ships and we have not committed to a specific date yet (I wouldn't want to publish something and then have it move).

Author: done, I replaced them with tbd.

>
> \*\* When .NET 6 releases, the goal is for the .NET SDK to be functional in version 16.11 for **downlevel** targeting. This means that you're not forced to update your SDK and Visual Studio versions simultaneously. However, you won't be able to target .NET 6 because of limitations in 6.0 features and C#10 features in version 16.11. This compatibility is specifically for targeting 5.0 and below.

> 

Reviewer: ```suggestion
>
```

> ^6.0.300 and newer SDKs will require a minimum VS version of 17.0

Reviewer: ```suggestion
> ^ 6.0.300 and newer SDKs require a minimum Visual Studio version of 17.0.
```
